src/clientSideScene/CameraControls.ts

```diff
@@ -63,19 +63,19 @@ interface ThreeCamValues {
 
 export type ReactCameraProperties =
   | {
-    type: 'perspective'
-    fov?: number
-    position: [number, number, number]
-    target: [number, number, number]
-    quaternion: [number, number, number, number]
-  }
+      type: 'perspective'
+      fov?: number
+      position: [number, number, number]
+      target: [number, number, number]
+      quaternion: [number, number, number, number]
+    }
```

Comment:
You may have unintentionally changed some formatting. I think that if you run biome from the `rust/` directory, it doesn't pick up the config. Could that be what happened?

Reply:
Nah, I ran format from the repo root and it made this change. I opened it as a PR: https://github.com/KittyCAD/modeling-app/pull/7354